modules/gdscript/gdscript_vm.cpp

```diff
@@ -116,6 +116,7 @@ String GDScriptFunction::_get_call_error(const Callable::CallError &p_err, const
 
 	if (p_err.error == Callable::CallError::CALL_ERROR_INVALID_ARGUMENT) {
 		int errorarg = p_err.argument;
+		ERR_FAIL_NULL_V_MSG(argptrs[errorarg], "Invalid argument type in " + p_where + ".", "CallError holds invalid pointers to arguments.");
```

Comment:
```suggestion
		ERR_FAIL_COND_V_MSG(errorarg < 0 || argptrs[errorarg] == nullptr, "Invalid CallError argument index or pointers in " + p_where + ".", "Invalid CallError argument index or pointers.");
```

Reply:
The rephrasing for the `MSG` seems good.

The reason I used the more generic "Invalid argument type in ..." message for the return value is that this is the type of call error we're handling here. The return value should be a message that relates to the bug in the user's script that caused the call error. And the `ERR_FAIL` message is for us developers.

Reply:
That being said if we consider this condition should never be hit, and thus doesn't need to give useful info about potential script errors, we can just go with something like "GDScript bug, please report" and the `MSG` giving details.

Reply:
Done.